lib/xcpretty/formatters/formatter.rb

```diff
@@ -173,10 +178,13 @@ def format_will_not_be_code_signed(message)
     end
 
     def format_other(text)
-      ""
+      if @unmatched and text and !text.empty? and !text.gsub(/\s+/, '').lstrip.empty?
```

Comment:
See my comment above regarding unmatched lines with leading whitespace.

Reply:
I'm not so "fan" to remove space leaded lines, example :

```
[appname] ▸ Processing product packaging  for app appname.app.xcent
unmatched: Entitlements:
unmatched: {
unmatched:     "com.apple.security.get-task-allow" = 1;
unmatched: }
```
will "hide" a crucial line... the unmatched is clearly intended to be able to "debug" xcpretty (maybe a better comment in the help could hint this behavior)

Reply:
got 10 lines on one project with "unmatched:     ",  and 26 on another project, all about Entitltments "json/plist output"

Reply:
Ok. We can leave it as is.